toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt

```diff
@@ -85,11 +87,13 @@ public fun MapView(
     graphicsOverlays: GraphicsOverlayCollection = rememberGraphicsOverlayCollection(),
     locationDisplay: LocationDisplay = rememberLocationDisplay(),
     wrapAroundMode: WrapAroundMode = WrapAroundMode.EnabledWhenSupported,
+    attributionBarVisible: Boolean = true,
```

Comment:
Should the parameter be suffixed with "is"?
```suggestion
    isAttributionBarVisible: Boolean = true,
```